### PR TITLE
Topbar

### PR DESCRIPTION
**Pull request by josepplloo**
## Summary

Navigation bar added
## Screenshots

topbar

## Trello Card
https://trello.com/c/BBvHRHVy/24-routing-ii-topbar

### CODE REVIEW

File: react/tic-tac-toe/src/app/screens/Podium/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import Matches from './components/Matches';
+import styles from './styles.module.scss';
+
+function Podium() {
+  return (
+    <div className={styles.container}>
```
Comment (CrisVega08) [inline react/tic-tac-toe/src/app/screens/Podium/index.js]: If you use this div inside of Matches component you can remove podium and use Matches.

File: react/tic-tac-toe/src/app/screens/Podium/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+.container {
+ display: flex;
```
Comment (CrisVega08) [inline react/tic-tac-toe/src/app/screens/Podium/styles.module.scss]: Here review the tab size.

File: react/tic-tac-toe/src/app/components/AppRouter/components/AuthInfo/styles.module.scss [hunk 3]
```diff
@@ -1,19 +1,40 @@
...
+  }
+}
+
+.link {
+  background-color: $black;
+  color: $white;
+  display: flex;
+  max-width: 80px;
+  margin: 0 5px;
+  overflow:auto;
```
Comment (CrisVega08) [inline react/tic-tac-toe/src/app/components/AppRouter/components/AuthInfo/styles.module.scss]: Add space here.

File: react/tic-tac-toe/src/app/components/AppRouter/components/AuthInfo/styles.module.scss [hunk 4]
```diff
@@ -1,19 +1,40 @@
...
-    border-color: $orange;
+  }
+}
+
+.link {
+  background-color: $black;
+  color: $white;
+  display: flex;
+  max-width: 80px;
+  margin: 0 5px;
```
Comment (CrisVega08) [inline react/tic-tac-toe/src/app/components/AppRouter/components/AuthInfo/styles.module.scss]: sort it.

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: react/tic-tac-toe/src/app/components/AppRouter/components/AuthInfo/styles.module.scss (hunk 4 above)
Comment (josepplloo) [reply react/tic-tac-toe/src/app/components/AppRouter/components/AuthInfo/styles.module.scss]: IDK what happen with that file (Don't update)

File: react/tic-tac-toe/src/app/screens/Podium/index.js [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
+import React, { Component } from 'react';
 import Spinner from 'react-spinkit';
 
-import getMatches from '../../../../../services/MatchesService';
+import getMatches from '~services/MatchesService';
 
 import styles from './styles.module.scss';
 
-class Matches extends Component {
+class Podium extends Component {
```
Comment (CrisVega08) [inline react/tic-tac-toe/src/app/screens/Podium/index.js]: If you change the name of component change also file name.

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: react/tic-tac-toe/src/app/screens/Podium/index.js (hunk 5 above)
Comment (josepplloo) [reply react/tic-tac-toe/src/app/screens/Podium/index.js]: Would you like me to change this name (use Matches label in app instead of Podium) ?

Comment (CrisVega08) [reply react/tic-tac-toe/src/app/screens/Podium/index.js]: Yes, but i refer to name of folder that this Component, look that folder has name Matches change to Podium.
! [image] (https://user-images.githubusercontent.com/26824183/62071967-3e37c880-b203-11e9-9fb7-7a1c85343160.png)

**Review by CrisVega08 — APPROVED** [review]
Only change folder name, nice work

**Pull request merged**